### PR TITLE
Kraken: Fix TestGetFuturesOrderbook and consolidate test pairs

### PR DESCRIPTION
**Pull request by thrasher-**
# PR Description

Fixes the TestGetFuturesOrderbook error and consolidates test pairs

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/kraken/kraken_test.go [hunk 1]
```diff
@@ -35,8 +35,12 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/portfolio/withdraw"
 )
 
-var k *Kraken
-var wsSetupRan bool
+var (
+	k               *Kraken
+	wsSetupRan      bool
+	futuresTestPair = currency.NewPairWithDelimiter("PF", "XBTUSD", "_")
+	spotTestSpair   = currency.NewPair(currency.XBT, currency.USD)
```
Comment (shazbert) [inline exchanges/kraken/kraken_test.go:42]:
```suggestion
	spotTestpair   = currency.NewPair(currency.XBT, currency.USD)
```

**Review by shazbert — CHANGES_REQUESTED** [review]
One thing from me

**Review by shazbert — APPROVED** [review]
Thanks, tACK.

**Review by gloriousCode — APPROVED** [review]
tACK!
This is an AI generated image of a kraken defeating an API error. Please enjoy
image

**Pull request merged**